### ISSUE TITLE
Relative Path to Global Bookmark

### ISSUE THREAD

**Issue by dylan-k** [estruyf/vscode-squarl#4]
I added a global bookmark. When I looked in my user settings.json (not the one for the project, the global one), the path to the file was relative, based on the project I was in when I made the bookmark. This means the global bookmark won't work in any other project, right? 

I tried entering the full path manually into the JSON setting, being careful to escape the slash characters, but then Squarl just thought it had a deleted bookmark.

**Comment by estruyf** [comment]
Hi @dylan-k, it is correct that bookmarks are stored relatively from the current project. It could indeed cause an issue for global bookmarks. 

This might require an enhancement with an extra question to store it as an absolute or relative path and make the file check work correctly.

**Comment by dylan-k** [comment]
Yes, the bookmarks are written as relative links, from the currently active project.

I don't understand the need for the extra question. Why would a global link use a relative path?

**Comment by estruyf** [comment]
Fair point! Changes will be implemented to always store the files as absolute paths fo global bookmarks.

**Comment by estruyf** [comment]
@dylan-k a new version got released to test out. Let me know if this solves your issue.
